plugin/src/main/java/net/dzikoysk/funnyguilds/feature/command/user/InviteCommand.java

```diff
@@ -42,14 +53,52 @@ public void execute(@CanManage User deputy, Guild guild, String[] args) {
         when(args.length < 1, this.messages.generalNoNickGiven);
         when(guild.getMembers().size() >= this.config.maxMembersInGuild, formatter.format(this.messages.inviteAmount));
```

Comment:
Mam wątpliwości czy jest to potrzebne, skoro niżej i tak jest sprawdzane łącznie z graczami w okolicy? W teorii szybsza wiadomość. W praktyce gracz używający komendy i tak otrzyma ten sam błąd natychmiast, bez żadnej różnicy z jego punktu widzenia, tylko w kodzie później — ale ten check niżej, to faktycznie ten realnie potrzebny.